src/components/snapshots/takeSnapshot.ts

```diff
@@ -1,12 +1,12 @@
-import fs from 'fs/promises'
-import fsSync from 'fs'
+import fs from 'fs'
```

Comment:
There was already two different imports of fs, I've just combined them into one rather than creating a third.